gnome-initial-setup/pages/language/cc-language-chooser.c

```diff
@@ -569,6 +578,7 @@ cc_language_chooser_constructed (GObject *object)
 
         if (priv->language == NULL)
                 priv->language = cc_common_language_get_current_language ();
+        priv->initial_language = g_strdup (priv->language);
 
         sync_all_checkmarks (chooser);
         show_more (chooser);
```

Comment:
Not in the new code added, but I think the return value of an int function is undefined if not specified.

Reply:
Sorry -- ignore that.  I missed expanding part of the code and mixed two functions together.